### PR TITLE
[13.0] [MIG] purchase_triple_discount: Migration to 13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT31773

please @chienandalu @victoralmau review it

This PR superseds https://github.com/OCA/purchase-workflow/pull/930

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]
Code review OK

File: purchase_triple_discount/report/report_purchase_order.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document"
+        inherit_id="purchase_discount.report_purchaseorder_document"
+    >
+        <xpath expr="//table[1]/thead/tr//th[last()]" position="before">
```
Comment (pedrobaeza) [inline purchase_triple_discount/report/report_purchase_order.xml]: Putting them at the end is not correct. You should put them after `discount`:

```suggestion
        <xpath expr="//th[@name='th_discount']" position="after">
```

**Comment by CarlosRoca13** [comment]
Change done @pedrobaeza

**Comment by pedrobaeza** [comment]
You have put them in the incorrect commit.

**Comment by CarlosRoca13** [comment]
Sorry, changed

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**